client_folder_react_app/ethisim/src/App.js

```diff
@@ -6,22 +6,22 @@ import {
   Route,
   Redirect,
 } from "react-router-dom";
+import Register from './pages/register_account';
 import Login from './pages/login';
-
 export default function App() {
   return (
     <Router>
         <Route exact path="/">
-          <Redirect to="/login" />
+          <Redirect to="/register_account" />
```

Comment:
I would rename register_account to /signup or /registration. Probably want to first redirect to login page first (rather than registration). And add a hyper link (below password box in /login, "Need to register an account?" that redirects to /signup); this can be done in another pr.